main/res/values/preference_keys.xml

```diff
@@ -128,6 +128,7 @@
     <string translatable="false" name="pref_settingsversion">settingsversion</string>
     <string translatable="false" name="pref_trackableaction">trackableaction</string>
     <string translatable="false" name="pref_shareafterexport">shareafterexport</string>
+    <string translatable="false" name="pref_includefoundstatus">includefoundstatus</string>
```

Comment:
Do not include translations in your patch, those are handled through crowdin.net.